packages/framework/collabspaces/src/contracts.ts

```diff
@@ -83,5 +94,27 @@ export interface IEfficientMatrixTest {
 	isAttached: boolean;
 
 	// Returns a structure with various debug info about the cell
-	getCellDebugInfo(row: number, col: number): Promise<{ channel?: ICollabChannelCore }>;
+	getCellDebugInfo(
+		row: number,
+		col: number,
+	): Promise<{ channel?: ICollabChannelCore; channelId: string; rowId: string; colId: string }>;
+
+	getReverseMapsDebugInfo(
+		rowId?: string | undefined,
+		colId?: string | undefined,
+	): ReverseMapsDebugInfoType;
+}
+
+export type ReverseMapType = "row" | "col";
+
+export interface IReverseMap {
```

Comment:
I believe this should not be here - this file is about public contracts.
Maybe in the future we would benefit from having "internal" contracts file (or multiple implementations of this interface), but for now I'd just move it into reverseMap.ts

Reply:
yes, moved to reverseMap.ts for now